### ISSUE TITLE
[feature request] Jupyter notebooks support

### ISSUE THREAD

**Issue by giovannibonaccorsi** [QL-Win/QuickLook#462]
Brief introduction: [Jupyter Notebooks](https://en.wikipedia.org/wiki/Project_Jupyter#Jupyter_Notebook) provides an interactive browser-based environment for programming (in origin with Python but now with many other languages) but at the end of the day they are plain [JSON files](https://ipython.org/ipython-doc/3/notebook/nbformat.html) with the extension `.ipynb` .

They're notoriously hard to consult without running a proper Python app or converting to another format. I've recently found some projects which allow to avoid all the overhead. One of these is a Quick Look plugin for MacOS: [ipynb quicklook](https://github.com/tuxu/ipynb-quicklook). 

I know nothing about development of plugins for QuickLook for Windows, but if converting from a MacOS plugin is easier than writing the plugin from scratch, it would be a much appreciated feature.

**Comment by OdinTech3** [comment]
I am also very interested in this. Is there documentation on how to build plugins for quicklook??

**Comment by xupefei** [comment]
I believe that this can be done by using https://github.com/kokes/nbviewer.js to render JSON to HTML and thereafter call the HtmlViewer to display the file.

The whole process will be similar to MarkdownViewer: https://github.com/QL-Win/QuickLook/blob/master/QuickLook.Plugin/QuickLook.Plugin.MarkdownViewer/Plugin.cs.

**Comment by xupefei** [comment]
@OdinTech3 Yes, there is: https://github.com/QL-Win/QuickLook/wiki/Integration-and-Development and https://github.com/QL-Win/QuickLook.Plugin.HelloWorld
